doc/authoring_help.md

```diff
@@ -28,7 +28,7 @@ To override help for a given command:
 
 ### Example YAML help file, \_help.py
 
-```py
+```python
```

Comment:
`py` is an alias of `python`.

https://github.com/github/linguist/blob/master/lib/linguist/languages.yml#L4367

```yaml
Python:
  ...
  extensions:
  - ".py"
```

But good to have them unified.